ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/privacy/PrivateTransactionGroupResult.java

```diff
@@ -47,6 +49,26 @@ public PrivateTransactionGroupResult(final PrivateTransaction tx) {
     this.privacyGroupId = tx.getPrivacyGroupId().get().toBase64String();
   }
 
+  @JsonCreator
+  public PrivateTransactionGroupResult(
```

Comment:
Do we need this change? It's not in the AT package, will this affect the priv_getPrivateTransaction rpc response?